src/coreclr/src/jit/codegenxarch.cpp

```diff
@@ -1769,7 +1769,7 @@ void CodeGen::genCodeForTreeNode(GenTree* treeNode)
             // Have to clear the ShadowSP of the nesting level which encloses the finally. Generates:
             //     mov dword ptr [ebp-0xC], 0  // for some slot of the ShadowSP local var
 
-            unsigned finallyNesting;
+            size_t finallyNesting;
```

Comment:
Might have been easier to just cast the gtVal1 in the next line to unsigned, for a smaller change.